### PR TITLE
Adding Background and Footer

### PR DESCRIPTION
**Pull request by ghost**
image

There is a note that it can't automatically merge.
I've synced and used git pull before.
Let me know how I can improve, or maybe it can be done on your side.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Resolved conflict
Improvements on the body, variable selector.

**Comment by samipak458** [comment]
Share screenshot of complete website, so I can review it properly.

**Comment by ghost** [comment]
It is not aligned well due to I used screenshot
github

Here is the codepen to make it more clear if you wanna inspect it further more.
https://codepen.io/DHCJS/pen/zYjjejL

**Comment by samipak458** [comment]
Make it responsive @DHCJS

File: index.html [hunk 1]
```diff
@@ -295,16 +296,21 @@
 
     .footer-col{
       width: 25%;
-      padding: 0 15px;
+      padding: 0 20px;
```
Comment (ghost) [inline index.html:299]: change the space on top

File: index.html [hunk 2]
```diff
@@ -279,13 +279,14 @@
...
 
       .footer {
-         padding: 50px 0;
+         
+         padding: 30px 0;
          font-family: 'Poppins', sans-serif;
        }
 
      .container-a{
-        max-width: 1170px;
```
Comment (ghost) [inline index.html]: Responsive

File: index.html [hunk 3]
```diff
@@ -295,16 +296,21 @@
...
-      padding: 0 15px;
+      padding: 0 20px;
       }
 
     .footer-col h4{
       font-size: 35px;
       color: black;
       margin-bottom: 20px;
       position: relative;
+      font-size: clamp(1.2rem, 2.5vw, 1.5rem);
```
Comment (ghost) [inline index.html:307]: Responsive

File: index.html [hunk 4]
```diff
@@ -295,16 +296,21 @@
...
     }
-
+
```
Comment (ghost) [inline index.html:309]: Responsive for the link text

**Review by ghost — COMMENTED** [review]
Making changes to Responsive Footer and Several changes

Please kindly check it @samipak458. Didn't quite catch what you mean last time. Sorry for that

image

**Comment by samipak458** [comment]
Shelf of book is overflowing, can you fix it? 
Also resolve the conflict

**Comment by ghost** [comment]
>Shelf of book is overflowing, can you fix it?

I'll pass on that at this moment. If it is not fixed in the next week, I'll maybe make an issue. I have another task to do at this moment.

**Comment by samipak458** [comment]
Ok no problem, you can create new issue and fix it.

**Pull request merged**